demos/notebooks/demo_pipeline.ipynb

```diff
@@ -64,18 +116,53 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "logging.basicConfig(format=\n",
-    "                          \"%(relativeCreated)12d [%(filename)s:%(funcName)20s():%(lineno)s] [%(process)d] %(message)s\",\n",
-    "                    # filename=\"/tmp/caiman.log\",\n",
-    "                    level=logging.WARNING)"
+    "movie_path = download_demo('Sue_2x_3000_40_-46.tif')\n",
+    "print(f\"Original movie for demo is in {movie_path}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you want to adapt this demo for your own data, just direct the `movie_path` variable to your own movie:\n",
+    "\n",
+    "    movie_path = 'full/path/to/your/movie.filetype'\n",
+    "\n",
+    "If you have a recording system that breaks up the data across multiple files, see [Appendix 2: Working with multiple files](#multiple_files). "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "    <h2 style=\"margin-top: 0;\">File types that Caiman can read</h2>\n",
+    "    While this demo uses a movie that has been stored in <i>tif</i> format, Caiman can handle movies in multiple common (and not so common) formats, including:\n",
+    "\n",
+    "    hdf5/h5, n5, zarr, avi, nwb, npz\n",
+    "    \n",
+    "</div>"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Select file(s) to be processed\n",
-    "The `download_demo` function will download the specific file for you and return the complete path to the file which will be stored in your `caiman_data` directory. If you adapt this demo for your data make sure to pass the complete path to your file(s). Remember to pass the `fname` variable as a list."
+    "## Load and visualize raw data\n",
+    "Caiman has a built-in movie class that you can use to view your movie. \n",
+    "\n",
+    "The movie object has many convenient features. Once you have loaded the movie (using `cm.load()`), you can view your raw data using `movie.play()`. This `play()` function has multiple parameters you can explore, including: \n",
```

Comment:
I'd remove the "The movie object has many convenient features" line as it's not saying anything

Reply:
Updating this section